From 439032e3256d9f9163d9ba6ccd8f5956856f7470 Mon Sep 17 00:00:00 2001
From: Quynh Nguyen <quynhnguyen.xb@renesas.com>
Date: Fri, 16 Oct 2020 08:58:48 +0700
Subject: [PATCH 461/475] dt-bindings: usb-xhci: Document R8A7742 support

Add USB-XHCI device tree binding for the R8A7742 (RZ/G1H) SoC.

Signed-off-by: Quynh Nguyen <quynhnguyen.xb@renesas.com>
---
 Documentation/devicetree/bindings/usb/usb-xhci.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/usb/usb-xhci.txt b/Documentation/devicetree/bindings/usb/usb-xhci.txt
index bf5192b..c040cc3 100644
--- a/Documentation/devicetree/bindings/usb/usb-xhci.txt
+++ b/Documentation/devicetree/bindings/usb/usb-xhci.txt
@@ -3,8 +3,8 @@ USB xHCI controllers
 Required properties:
   - compatible: should be one of "generic-xhci",
     "marvell,armada-375-xhci", "marvell,armada-380-xhci",
-    "renesas,xhci-r8a7790", "renesas,xhci-r8a7791", "renesas,xhci-r8a7743",
-    "renesas,xhci-r8a7744"
+    "renesas,xhci-r8a7790", "renesas,xhci-r8a7791", "renesas,xhci-r8a7742",
+    "renesas,xhci-r8a7743","renesas,xhci-r8a7744"
     (deprecated: "xhci-platform").
   - reg: should contain address and length of the standard XHCI
     register set for the device.
-- 
2.7.4

